views/head.jade

```diff
@@ -7,15 +7,15 @@ link(rel='icon', type='image/png', href='/images/icon.png')
 
 // Stylesheets too include
 link(rel='stylesheet', href="https://fonts.googleapis.com/css?family=Open+Sans|Russo+One|Ubuntu+Mono")
-link(rel='stylesheet', href="//cdnjs.cloudflare.com/ajax/libs/normalize/2.1.3/normalize.min.css")
-link(rel='stylesheet', href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.0.0-rc2/css/bootstrap#{min}.css")
+link(rel='stylesheet', href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.2/normalize.min.css")
```

Comment:
`#{min}` would automatically add `.min` to the file name when specified at build time, which was a handy feature for debugging and releasing.